acceptance/lib/pxp-agent/test_helper.rb

```diff
@@ -440,17 +440,17 @@ def get_process_pids(host, process)
   pids
 end
 
-def wait_for_sleep_process(target)
+def wait_for_sleep_process(target, seconds_to_sleep)
   begin
     ps_cmd = target['platform'] =~ /win/ ? 'ps -efW' : 'ps -ef'
-    sleep_process = target['platform'] =~ /win/ ? 'PING' : '/bin/sleep'
+    sleep_process = target['platform'] =~ /win/ ? 'PING' : " /bin/sleep #{seconds_to_sleep}"
```

Comment:
Why the space before the `/bin/sleep`?

Reply:
To match only `/bin/sleep` rather than also `/usr/bin/sleep`.

Reply:
ah, ok